### ISSUE TITLE
 unexpected 'array' (T_ARRAY), expecting function (T_FUNCTION) or const (T_CONST) in MultipartFormDataModule.php on line 20

### ISSUE THREAD

**Issue by Paul-Lebranchu** [gotenberg/gotenberg-php#7]
Hi,

I've tried to convert an HTML page into a PDF but I already got the same error:

"unexpected 'array' (T_ARRAY), expecting function (T_FUNCTION) or const (T_CONST) in MultipartFormDataModule.php on line 20"

and i don't understand why.

Here is my PHP code:
```php
  <?php
    require 'vendor/autoload.php';
    use Gotenberg\Gotenberg;
    use Gotenberg\Stream;
    $apiUrl = "http://localhost:3000";
    $request = Gotenberg::chromium($apiUrl)->html(Stream::path('etiquette.html'));
    ?>
```

I've tried with other method as:

```php
use Gotenberg\Gotenberg;

   // Converts a target URL to PDF and saves it to a given directory.
    $filename = Gotenberg::save(
    Gotenberg::chromium($apiUrl)->url('https://my.url'), 
    $pathToSavingDirectory
);
```

but I've got the same result.

Have you got any idea about why I've got this error please?

Thanks

**Comment by gulien** [comment]
Hello @Paul-Lebranchu,

What's your PHP version?

**Comment by Paul-Lebranchu** [comment]
It's PHP Version 7.2.33

**Comment by gulien** [comment]
That's why 😄 This library is only available for PHP >= `7.4`. Consider upgrading to a maintained version of PHP 👍

**Comment by Paul-Lebranchu** [comment]
Ok! Thanks for your answer, have a nice day :)
